Sources/VersionChecker/Networking/ApiFetcher+Extension.swift

```diff
@@ -22,7 +22,9 @@ import InfomaniakCore
 
 extension Endpoint {
     static func version(store: Store, platform: Platform, appName: String) -> Endpoint {
-        return Endpoint(path: "/1/app-information/versions/\(store.rawValue)/\(platform.rawValue)/\(appName)")
+        return Endpoint(path: "/1/app-information/versions/\(store.rawValue)/\(platform.rawValue)/\(appName)", queryItems: [
+            URLQueryItem(name: "only", value: "platform,min_version,name,published_versions.tag,published_versions.tag_updated_at,published_versions.build_version,published_versions.build_min_os_version")
```

Comment:
We do not need `platform` or `name`.
Currently, we still need `published_versions.type` but we could manage to remove it from or model.

The properties listed here should be the ones in the `Version` and `PublishedVersion` models so Swift can decode the JSON correctly 👍